extended-utxo-spec/extended-utxo-specification.tex

```diff
@@ -796,47 +794,50 @@ \subsection{The definition of EUTXO-2}
 \begin{ruledfigure}{H}
   \center{
   \begin{tabular}{rll}
-  \s{CurrencyID}&: & an identifier for a custom currency. This is an alias
-  for the \s{Address} type\\
-                & & from Figure~\ref{fig:basic-types}.\\
+    \currency &: & an identifier for a custom currency. This is an alias
+                for the \s{Address} type\\
+             &  & from Figure~\ref{fig:basic-types}.\\
+    \\
+    \nativeCur &: & a distinguished \currency{} representing the native currency of the ledger.\\
+    \\
+    \token   &: & a type consisting of identifiers for individual tokens.  These will probably\\
+             &  & be hashes in practice.\\
     \\
-    \s{Token}&:& a type consisting of identifiers for individual tokens.  These will probably\\
-               & &be hashes in practice.\\
-\\
-    \qtymap &=& \Map{\s{CurrencyId}}{\Map{\s{Token}}{\s{Quantity}}}\\
+    \nativeTok &: & a distinguished \token{} representing the currency token of the native currency.\\
     \\
-  $\qtymap^{\pm}$ &=& \Map{\s{CurrencyId}}{\Map{\s{Token}}{\s{Quantity}^{\pm}}}\\
+    \qtymap   &=& \FinSup{\currency}{\FinSup{\token}{\qty}}\\
+    \\
+    \qtymappm &=& \FinSup{\currency}{\FinSup{\token}{\qtypm}}\\
```

Comment:
Ah, I should say that FSFs have to be to a monoid (otherwise you don't have a zero!). FSFs form a monoid, so then it just works.

But yes, I'll add sentence.

Reply:
Oh, I already said that. But I'll add something about the nested function.